admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/css/admin/entityForm.css

```diff
@@ -263,6 +263,7 @@ button:disabled:hover, button.disabled:hover {
 .filter-info {
     float: right;
     text-align: right;
+    top: -5px;
```

Comment:
I think this change will make it "higher" when filter is not applied
I think the issue is here that once filter is applied filter button with "X" button are wrapped into div with style .filter-info .button-group {
    margin-top: 6px;
}
.filter-info > .filter-button {
    margin-top: 6px;
}
that does this 6px offset. So we can remove it at all, or wrap search elements also in some kind of this thing, or just add new style when filter is applied etc if we want to have them in synch

@StanislavFedorov can you take it from there ?

Reply:
@oleksiimiroshnyk 
I agree that better to remove this
```
.filter-info .button-group {
    margin-top: 6px;
}
```
but before this please @MarekB01 go through the admin UI and check if we have this button-group in other places

Reply:
@StanislavFedorov Seems like this button-group is only filter related. Also I tried to remove
.filter-info .button-group {
    margin-top: 6px;
}
But it didn't help

Reply:
@MarekB01 
**Also I tried to remove .. But it didn't help**

Do you mean this css? For me removing exactly this css block works

```
.filter-info .button-group {
    margin-top: 6px;
}
```

Reply:
@StanislavFedorov now it works for me too